spec/jobs/trs_mentor_sync_job_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe TRSMentorSyncJob, type: :job do
+  let!(:mentor_1) { create(:mentor, first_name: "Joe", last_name: "Bloggs") }
+  let!(:mentor_2) { create(:mentor, first_name: "Agatha", last_name: "Christie") }
+
+  before do
+    mentor_1
+    mentor_2
+  end
```

Comment:
Since you added the bang, I think you can delete these lines, right?

Reply:
apparently so, resolved in [ceb4cb0](https://github.com/DFE-Digital/itt-mentor-services/pull/302/commits/ceb4cb07e877b469bede08f4eca4abb81db26377)